nautobot_ssot/integrations/ipfabric/constants.py

```diff
@@ -22,3 +22,4 @@
 SAFE_DELETE_LOCATION_STATUS = CONFIG.get("ipfabric_safe_delete_location_status", "Decommissioning")
 SAFE_DELETE_VLAN_STATUS = CONFIG.get("ipfabric_safe_delete_vlan_status", "Deprecated")
 SAFE_DELETE_IPADDRESS_STATUS = CONFIG.get("ipfabric_safe_delete_ipaddress_status", "Deprecated")
+LAST_SYNCHRONIZED_CF_NAME = "last_synced_from_ssot"
```

Comment:
IMO I think it should be something that we should utilize across the integrations as it's been extremely helpful with client engagements being able to determine which SoR some object came from when you have multiple SSoTs operating together. This might be moot point though with some of the changes coming in core Nautobot though?